app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+  <%= f.label :name %><br />
+  <%= f.text_field :name %>
+</div>
+
```

Comment:
- [ ] User should be able to login with only email and password. Kindly remove this name field

Reply:
Fixed.